src/tagger.jl

```diff
@@ -0,0 +1,225 @@
+"""
+This contains the module "PerceptronTagger" which is uses model in AveragePerceptron
+In this training can be done and weights can be saved
+Or a pretrain weights can be used (which are trained on same features)
+and train more or can be used to predict
+
+DataStructures and Random are essential dependencies
+JLD is needed when weights are to be saved
+
+To train:
+PerceptronTagger.train([[("today","NN"),("is","VBZ"),("good","JJ"),("day","NN")]])
+
+To predict tag:
+PerceptronTagger.test(["today", "is"])
+"""
+
+module PerceptronTagger()
+    using DataStructures
+    using JLD
+    using Random
+
+    include("averagePerceptron.jl")
+    import .AveragePerceptron
+
+    mutable struct init

```

Comment:
`init` is a bad name for a type

Reply:
okay, I will change that. And I will both the files into one.

Reply:
Sir, I just read somewhere that I can't make member functions with struct in julia.
So I should use module for member functions.
is it true or is there a way to make member functions without using modules??

Reply:
I have changed the `tagger.jl` to use BSON and have committed it . But currently is showing some checks are pending . Please have a look and tell whether are correct or not.